types/redux-form/lib/reduxForm.d.ts

```diff
@@ -34,7 +34,7 @@ export interface Config<FormData extends DataShape, P, S> {
      * the name of your form and the key to where your form's state will be
      * mounted under the redux-form reducer
      */
-    form: string;
+    form?: string;
```

Comment:
Q: in this [document ](http://redux-form.com/6.6.3/docs/api/ReduxForm.md/) it seems that form should be required? let me know if I misunderstand something here